test/integration/crud/insert.test.js

```diff
@@ -4,7 +4,6 @@ const { format: f } = require('util');
 const { expect } = require('chai');
 
 const Script = require('vm');
-const { normalizedFunctionString } = require('bson/lib/parser/utils');
```

Comment:
what's the deal with this function and the corresponding tests?

Reply:
We used to "normalize" function strings in 4.x BSON (remove a space after the keyword `function`) we no longer do that so we don't need this helper, plus that import doesn't exist anymore either, we only have a single bundle file now